gpslogger/src/main/java/com/mendhak/gpslogger/common/AppSettings.java

```diff
@@ -38,6 +38,7 @@
 
     @Override
     public void onCreate() {
+        Systems.setAppTheme(PreferenceHelper.getInstance().getAppThemeSetting());
```

Comment:
I thought moving this as early as possible could help with the brief white flash, when user has chosen dark only theme.  But alas... anyway I'll leave this here for now